### PR TITLE
docs: Update README

### PR DESCRIPTION
**Pull request by craigshoemaker**
- Clarify auth emulation port
- General text clean up

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -134,44 +136,62 @@ If you need to override the default values, provide the following options:
 
 ## Local authentication & authorization emulation
 
-When requesting `./auth/login/<provider>`, you will access a local authentication UI allowing you to set fake user information.
+The CLI allows you to mock and read authentication & authorization credentials.
+
+> **NOTE:** You must use port `4280` to access the authentication UI.
```
Comment (manekinekko) [inline readme.md:141]: While `4280` is the default port, users may provide a different port using the `--port` option.

File: readme.md [hunk 2]
```diff
@@ -134,44 +136,62 @@ If you need to override the default values, provide the following options:
...
+
+### Mocking credentials
+
+When requesting `http://localhost:4280/.auth/login/<PROVIDER_NAME>`, you have access a local authentication UI allowing you to set fake user information.
+
+Provider names available include:
```
Comment (manekinekko) [inline readme.md:147]: User may also use their own auth providers

Comment (craigshoemaker) [reply readme.md:147]: okay - maybe just take this section out then?

File: readme.md (hunk 1 above)
Comment (craigshoemaker) [reply readme.md:141]: I'll update the note to say that.

Comment (craigshoemaker) [inline readme.md:141]:
```suggestion
> **NOTE:** The default port is to access the authentication UI is `4280`, but you can override using the `--port` option.
```

File: readme.md [hunk 3]
```diff
@@ -134,44 +136,62 @@ If you need to override the default values, provide the following options:
...
+
+- aad
+- facebook
+- github
+- google
+- twitter
```
Comment (craigshoemaker) [inline readme.md:153]:
```suggestion
```

**Review by craigshoemaker — COMMENTED** [review]
Updated text based on review feedback.

**Review by manekinekko — APPROVED** [review]

**Comment by manekinekko** [comment]
Thank you @craigshoemaker 🎉

**Pull request merged**